jsapp/js/account/stripe.api.ts

```diff
@@ -47,11 +48,13 @@ export async function changeSubscription(
   });
 }
 
-export async function getOrganization() {
-  return fetchGet<PaginatedResponse<Organization>>(endpoints.ORGANIZATION_URL, {
-    errorMessageDisplay: t("Couldn't get data for your organization."),
-  });
-}
+export const useOrganizationQuery = () => useQuery({
+  queryFn: async () => {
+    const response = await fetchGet<PaginatedResponse<Organization>>(endpoints.ORGANIZATION_URL);
```

Comment:
Do we need `PaginatedResponse` here? 🤔

Reply:
Yes, it was there before, and implementation does take first element from an array.